src/store/reducers/org.reducer.ts

```diff
@@ -0,0 +1,36 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { OrgProfileRes } from 'src/core/adaptors';
+
+export interface UserState {
+  orgProfile: OrgProfileRes;
+  isAuthenticated: boolean;
+}
+
+const initialState: UserState = {
+  orgProfile: {
+    imageUrl: '',
+    did: '',
```

Comment:
also, I need an `org id` (not did) to get org API (organization profile).
other properties don't matter, you can remove them because, in the organization profile route for every reloading, I should call org API (get).